BBallStatsTrack/src/com/example/bballstatstrack/models/Player.java

```diff
@@ -0,0 +1,265 @@
+package com.example.bballstatstrack.models;
+
+public class Player
+{
+    private static final String B_BALL_STAT_TRACK = "BBallStatTrack";
+
+    private int mNumber;
+
+    private String mFullName;
+
+    private int m2ptFGMiss = 0;
+
+    private int m2ptFGMade = 0;
+
+    private int m3ptFGMiss = 0;
+
+    private int m3ptFGMade = 0;
+
+    private int mFTMiss = 0;
+
+    private int mFTMade = 0;
+
+    private int mOffRebound = 0;
+
+    private int mDefRebound = 0;
+
+    private int mAssist = 0;
+
+    private int mTurnover = 0;
+
+    private int mSteal = 0;
+
+    private int mBlock = 0;
+
+    private int mFoul = 0;
+
+    private int mPlayingTimeSec = 0;
+
+    public Player( int id, String fullName )
+    {
+        setNumber( id );
+        setFullName( fullName );
+    }
+
+    public Player( int number, String fullName, int miss1pt, int miss2pt, int miss3pt, int made1pt, int made2pt,
```

Comment:
See this for an example :http://www.javaworld.com/article/2074938/core-java/too-many-parameters-in-java-methods-part-3-builder-pattern.html

Sadly BMS seems to be down ill paste the link regarding Builder pattern in Effective Java the site is again active.